discovery-package/src/main/resources/zosmf-static-definition.yaml.template

```diff
@@ -9,7 +9,7 @@ services:
       description: 'IBM z/OS Management Facility REST API service. Once configured you can access z/OSMF via the API gateway: https://${ZOWE_EXPLORER_HOST}:${GATEWAY_PORT}/ibmzosmf/api/v1/info'
       catalogUiTileId: zosmf
       instanceBaseUrls:
-        - ${ZWE_zOSMF_scheme:-https}://${ZOSMF_HOST}:${ZOSMF_PORT}/
+        - %ZOSMF_SCHEME%://${ZOSMF_HOST}:${ZOSMF_PORT}/"
```

Comment:
The configuration is after ZWE fullfil the other values and therefore it is needed to use %, right?

Have you considered to update ZWE code which propagate those values?

Reply:
Correct. I've considered it, but there's nothing like that currently in install-packaging, and it's specific for discovery service. I believe this is the right place.
If the preConfigure is enabled it will be much easier because it can be solved by just exporting `ZWE_zOSMF_scheme` environment variable.